packages/dd-trace/src/plugins/util/test.js

```diff
@@ -16,14 +16,15 @@ const {
   GIT_COMMIT_AUTHOR_EMAIL,
   GIT_COMMIT_AUTHOR_NAME,
   GIT_COMMIT_MESSAGE,
-  CI_WORKSPACE_PATH
+  CI_WORKSPACE_PATH, CI_PIPELINE_URL
```

Comment:
nitpick: to keep consistency 
```suggestion
  CI_WORKSPACE_PATH,
  CI_PIPELINE_URL
```